core/examples/luigi-core-material-ui/src/components/ProfileMenu.js

```diff
@@ -0,0 +1,144 @@
+import { useEffect, useState } from 'react';
```

Comment:
```suggestion
import React, { useEffect, useState } from 'react';
```
Same here, `React` should be kept.